pkg/oc/cli/describe/printer.go

```diff
@@ -62,7 +62,7 @@ var (
 
 	hostSubnetColumns          = []string{"NAME", "HOST", "HOST IP", "SUBNET"}
 	netNamespaceColumns        = []string{"NAME", "NETID"}
-	clusterNetworkColumns      = []string{"NAME", "NETWORK", "HOST SUBNET LENGTH", "SERVICE NETWORK", "PLUGIN NAME"}
+	clusterNetworkColumns      = []string{"NAME", "CLUSTER NETWORKS", "SERVICE NETWORK", "PLUGIN NAME"}
```

Comment:
Now the output of `oc get clusternetworks` is different, do we need to worry about backward compatibility issues here?
@openshift/api-review ^^

Reply:
@smarterclayton says that you can change `oc get` tables and columns.